src/main/java/taxiapp/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,37 @@
+package taxiapp.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxiapp.lib.Injector;
+import taxiapp.model.Car;
+import taxiapp.model.Driver;
+import taxiapp.service.CarService;
+import taxiapp.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxiapp");
+    private final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/driver/add.jsp").forward(req,resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = Long.valueOf(req.getParameter("driverId"));
+        Long carId = Long.valueOf(req.getParameter("carId"));
+        Car car = carService.get(carId);
+        Driver driver = driverService.get(driverId);
+        carService.addDriverToCar(driver, car);
+        req.getRequestDispatcher("/WEB-INF/views/car/driver/add.jsp").forward(req, resp);
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/cars/drivers/add");
```
let's use sendRedirect here instead